contents/docs/api/post-only-endpoints.mdx

```diff
@@ -52,6 +53,8 @@ curl -v -L --header "Content-Type: application/json" -d '{
 }' https://app.posthog.com/batch/ 
 ```
 
+The `historical_migration` field must be set to `true` when running imports from other systems. Events tagged with this field will be processed separately to avoid affecting real-time events.
```

Comment:
Maybe call out the benefit to the user: we are less likely to drop your events & they will be processed in the right order